run-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+export VMCK_URL='http://10.66.60.1:8000'
+
+function clean {
+    vagrant destroy -f || echo "vagrant destroy failed, but we don't care"
+}
+trap clean  EXIT
+
+set -ex
```

Comment:
Now that the file is so clean, you can move this to the top, even write `#!/bin/bash -ex` ;)